CONTRIBUTING.md

```diff
@@ -1,5 +1,41 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support!
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+Please note that discussions should be kept in English. Don't be afraid, if you don't speak English fluently. You can use [DeepL](https://www.deepl.com) which might be able to translate your language to English.
```

Comment:
"Don't be afraid" feels rather odd. I'd just say "If you don't speak English fluently you can use DeepL"

Reply:
https://github.com/jamulussoftware/jamuluswebsite/pull/157#issuecomment-740173966

We might have to rethink this